Build-Core.cmd

```diff
@@ -2,7 +2,7 @@
 setlocal
 
 SET CMDHOME=%~dp0.
-SET BUILD_FLAGS=/m:1 /v:m /fl /flp:logfile=build.log;verbosity=detailed
```

Comment:
I just checked and we can have /bl and /fl so binlog and normal log also generated.

@jdom would you prefer to add it back and write text log along the binary too with the new naming semantics?